patches/tModLoader/Terraria/release_extras/LaunchUtils/InstallNetFramework.sh

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env bash
+#Authors: covers1624, DarioDaf, Solxanich
+# Provided for use in tModLoader deployment. 
+
+#
+# THIS SCRIPT EXPECTS AS INPUTS:
+#   dotnet_dir     : Folder where the versions of dotnet will be extracted
+#   install_dir    : Target folder for the install (usually "$dotnet_dir/$version")
+#   dotnet_version : Version of dotnet to install
+#
+
+#chdir to path of the script and save it
+cd "$(dirname "$(realpath "$0")")"
+. ./BashUtils.sh
+
+#Cut everything before the second dot
+channel=$(echo "$dotnet_version" | cut -f1,2 -d'.')
+
+echo "Checking for old .NET versions to remove from folder"
+#If the dotnet dir exists, we need to do some cleanup
+if [ -d "$dotnet_dir" ]; then
+  # Find all folders inside the dotnet dir that don't match our target version and nuke it
+  for folder in $(ls "$dotnet_dir"); do
```

Comment:
[Iterating over ls output is fragile](https://github.com/koalaman/shellcheck/wiki/SC2045), use `for folder in "$dotnet_dir/*"; do` instead

Reply:
Tried this. 
Didn't work as expected - the any character didn't expand to search all items, and instead searched literal.

I'm comfortable keeping this fragility as the average person doesn't have reason to put stuff in there. 
"6.0.0" doesn't create issues afaik, and cleaning up old versions with same naming convention should be fine as well. 
For this one, if it ain't broke, and the fix is broke, don't spend time fixing the fix